src/main/java/com/jagrosh/jmusicbot/commands/MusicCommand.java

```diff
@@ -68,7 +68,7 @@ protected void execute(CommandEvent event)
             if(current==null)
                 current = settings.getVoiceChannel(event.getGuild());
             GuildVoiceState userState = event.getMember().getVoiceState();
-            if(!userState.inVoiceChannel() || userState.isDeafened() || (current!=null && !userState.getChannel().equals(current)))
+            if(!userState.inVoiceChannel() || userState.isDeafened() || (current!=null && !userState.getChannel().equals(current)) || userState.getGuild().getAfkChannel().equals(userState.getChannel()))
```

Comment:
CodeFactor seems to not like the complexity of the function after I added the AFK channel check in this if-statement

Reply:
This probably shouldn't be a simple addition to this if-statement anyway, as the error message will be incorrect. As you've written this, if a user is in an AFK channel, the bot will reply "You must be listening in NameOfAFKChannel to use that!" instead of something along the lines of "You cannot use that while in an AFK channel!"

Reply:
Alright, I wrote a message that's similar to [MusicCommand.java#L56](https://github.com/jagrosh/MusicBot/blob/master/src/main/java/com/jagrosh/jmusicbot/commands/MusicCommand.java#L56)
> You cannot use that command in an AFK channel!